README.md

```diff
@@ -1,2 +1,69 @@
 # mr_roboto
 A fun IRC bot that help us on work days
+
+# Development
+
+Prior to start the development on mr_roboto it's necessary to follow some steps
+in order to setup the environment: [*virtual environment*][pipenv], [*database
+migration*][alembic] and [*pre-commit hooks*][pre-commit].
+
+## Virtual environment through [**Pipenv**][pipenv]
+
+The first thing you need to do after the `git clone` is setup the *virtual
+environment* using **pipenv** tool. This tool requires you have at least
+*Python3* (used in this project) and **Python3-pip** (or install **pipenv**
+directly through your system's package manager).
+
+```
+$ pip install --user pipenv
+$ cd <cloned-repo-folder>
+$ pipenv --python python3
+$ pipenv shell
+$ pipenv install --dev
+```
+
```

Comment:
Well pointed for the two first paragraphs, the last one I use `pipenv shell` to actually not type `pipenv run` to every command 😁.. it's more a matter of taste and because of that I think we can highlight both ways and choose one to follow on the documentation. I'm ok with `pipenv run`.